components/HomeScreen.js

```diff
@@ -0,0 +1,53 @@
+
+import React, { useEffect, useState } from "react";
+import styled, { withTheme } from "styled-components/native";
+import backgroundImage from "../assets/background5.jpg";
+import {
+  TouchableOpacity, 
+  Text,
+} from "react-native";
+
+
+const HomeContainer = styled.ImageBackground`
+  flex: 1;
+  align-items: center;
+  justify-content: center;
+  padding: 20px;
+`;
+
+const HomeText = styled.Text`
+  font-size: 40px;
+  color:white;
+  font-weight:bold;
+  width:90%;
+`;
+const View = styled.TouchableOpacity`
+  padding: 20px;
+  background-color: #B250A1;
+  border-radius:50;
```

Comment:
We have to remember about units, there is nothing like `border-radius: 50` - you probably meant `border-radius: 50%`